### ISSUE TITLE
Add a dark mode

### ISSUE THREAD

**Issue by STEdirisuriya** [fossasia/loklak_search#631]
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**
No dark mode 

**Expected behavior:**
Add a dark mode and light mode.

**Comment by 0x48piraj** [comment]
As nothing is said about the issue and also the time limit is crossed, I'm taking this up! :+1:

**Comment by 0x48piraj** [comment]
Yes, @jajodiaraghav I'm working on this, please wait for few days and if a PR does not comes up, I'll notify you to claim this, sounds good ?

**Comment by ajayns** [comment]
Do you think this will actually be a useful feature? 
Opinions @ParthS007 @hemantjadon @singhpratyush @mariobehling

**Comment by praveenojha33** [comment]
@0x48piraj I want to implement themes for loklak search that will cover this issue.Please tell status of your work.I want to work on this issue.

**Comment by 0x48piraj** [comment]
It's a void issue @praveenojha33 
@STEdirisuriya talk to the maintainers and close it.

**Comment by championpaddler** [comment]
@praveenojha33  Instead we can create theme for loklak. Claiming this issue.

**Comment by championpaddler** [comment]
@simsausaurabh I am working on it.

**Comment by kpulkit29** [comment]
@praveenojha33  Are u still working on it?

**Comment by simsausaurabh** [comment]
This issue is free, no one is working on it.
